<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseNote</class>
 <widget class="QWidget" name="DatabaseNote">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="3" column="0">
    <widget class="QPushButton" name="add_to_table_button">
     <property name="text">
      <string>Добавить вещество</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="matter_name_label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>160</width>
       <height>20</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>160</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>Название вещества</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="group_name_label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>160</width>
       <height>20</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>160</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>Группа вещества</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="mpc_label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>160</width>
       <height>20</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>160</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>ПДК</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="matter_name_line_edit">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="matter_group_line_edit">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="mpc_double_spin_box">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="maximum">
      <double>9999999.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
